ci.sh

```diff
@@ -55,7 +55,6 @@ case $HOST_TARGET in
     ;;
   i686-pc-windows-msvc)
     MIRI_TEST_TARGET=x86_64-unknown-linux-gnu run_tests
```

Comment:
I was wondering about these in my ui test PR. Maybe instead of running this target twice (once on the host and once here( we could run the 32bit linux here instead of on the 64 bit linux host

Reply:
Ensuring the Linux target works on a Windows host is non-trivial because of file path handling (being a Linux target, we do support file system access). And the 64bit one is interesting because this is a 32bit host running a 64bit target. So I'd rather keep it like that.